### PR TITLE
Accessibility fixes for generated documentation

### LINKED ISSUE #396: Improve accessibliity of generated output
We are integrating spectaql within our product and while testing ran an accessibility review. It came back with a few small issues and one of our developers has written a pull request to fix them so creating an issue to track it.

### PR DESCRIPTION
**Pull request by simoncoggins**
Fixes #396

### CODE REVIEW

**Comment by newhouse** [comment]
@benogle what do you think about this?

File: src/themes/default/stylesheets/main.scss [hunk 1]
```diff
@@ -387,7 +387,7 @@ $background-arguments: #fafbfc;
     .operation-description > h5,
     .definition-description > h5,
     .definition-properties > h5 {
-      display: none;
+      @include visually-hidden();
```
Comment (benogle) [inline src/themes/default/stylesheets/main.scss:390]: Looks like this one is in the `spectaql` theme as well. Can you add that one and test it too?

Comment (newhouse) [reply src/themes/default/stylesheets/main.scss:390]: (meaning `src/themes/spectaql/*`)

**Comment by jackrhumphrey** [comment]
Hi I updated the spectaql theme and changed back the ::after text

Is "All topics" ok for the sr only text? Seems like you've gone that way with the "Close menu" aria-label

Thanks

**Comment by newhouse** [comment]
bump @benogle

**Review by benogle — APPROVED** [review]

**Comment by benogle** [comment]
> Is "All topics" ok for the sr only text? Seems like you've gone that way with the "Close menu" aria-label

Yep, all good 👍

**Comment by jackrhumphrey** [comment]
Thank you we were wondering if you had a timeline for when you will release those changes?

**Pull request merged**